### DISCUSSION TITLE
Podcast stream and feed overview

### DISCUSSION THREAD

**Discussion by RandomBloke85** [advplyr/audiobookshelf#1861] [Ideas]
Hello

Thank you for developping this fantastic app!

What I would like to see is the possibilty to just stream a podcast from its original source. I don't want to download anything on my server. What I miss is an overview from all podcast episodes in a feed. Maybe I'm doing it wrong.

Thank you.

**Comment by advplyr** [comment]
Hey, I'm not sure this is going to be supported. 

The original intention was for Abs to manage and serve the content on your server. Abs just provides a convenient way for you to source those audio files from RSS feeds.

This would require a significant rewrite of the data model. Not to say this would never be added in the future, but right now it seems outside the scope of this project.

You are probably better off with a regular podcast app if you don't want to store the episodes.

**Reply by RandomBloke85** [reply]
Alright, thank you. However, it's a beautiful app!
